HISTORY.rst

```diff
@@ -2,6 +2,32 @@
 History
 =======
 
+v0.2.0 (unreleased)
+-------------------
+Contributors to this version: Trevor James Smith (:user:`Zeitsperre`), Gabriel Rondeau-Genesse (:user:`RondeauG`).
```

Comment:
These sphinx-like directives (something I adapted/modified from Pangeo) require some code additions to the documentation configuration. If it's fine with you, I'll add them in this PR either today or tomorrow.

Reply:
No problem!